.vscode/launch.json

```diff
@@ -0,0 +1,70 @@
+{

```

Comment:
this file should be removed

Reply:
Sorry, I've been trying out vscode and it must've dropped those files in there when I turned on syntax highlighting for Django.  Removed and updated the .gitignore